docs/source/snowflake_usage.rst

```diff
@@ -6,11 +6,12 @@ For ``SnowflakeCompare``
 - ``on_index`` is not supported.
 - Joining is done using ``EQUAL_NULL`` which is the equality test that is safe for null values.
 - Compares ``snowflake.snowpark.DataFrame``, which can be provided as either raw Snowflake dataframes
-or the as the names of full names of valid snowflake tables, which we will process into Snowpark dataframes.
+  or the as the names of full names of valid snowflake tables, which we will process into Snowpark dataframes.
```

Comment:
small typo: "the as the names" -> "as the names"

Reply:
fixed. TY!